src/schema/types/models/Portfolio.ts

```diff
@@ -63,22 +63,15 @@ export const Portfolio = objectType({
 export const PortfolioSkill = objectType({
   name: "PortfolioSkill",
   definition(t) {
-    t.field("skill", {
-      type: "Skill",
-      resolve: () => {
-        return "Typescript or something";
-      },
-    });
+    t.string("name");
     t.int("level");
   },
 });
 
 export const PortfolioProject = objectType({
   name: "PortfolioProject",
   definition(t) {
-    t.field("project", {
-      type: "Project",
-    });
+    t.int("id");
     t.int("order");
```

Comment:
기존에 있던 대로 type을 project로 두어야 할 것 같아요. 그래야 project의 정보들을 불러올 수 있거든요 :D
```suggestion
    t.field("project", {
      type: "Project",
    });
    t.int("order");
```

Reply:
다음과 같이 수정한 후 실행을 한 결과, `skill`처럼 `null`이 돌아올 뿐이였습니다.
그래서 프로젝트 관련된 값들을 resolve로 작성하는 것이 나을 것 같다고 생각합니다.

어떻게 생각하시나요? 만약 긍정적으로 검토한다면 어떤 값들이 추가 됐으면 하는지 말씀드리면 좋겠습니다.

Reply:
해당 파일 32번째 라인 보니 타입 지정이 잘못되어있었네요! 이 부분만 수정하면 바로 될거 같습니다 :)
```
    t.list.field("projects", {
      type: "Project",
      resolve: async (root, _, __) => {
        return (await PortfolioService.getPortfolio(root.id)).PortfolioProject;
      },
    });
```

Reply:
제가 실수를 했었네요, 죄송합니다.
코멘트 내용을 반영하여 커밋하였습니다.